Library/Homebrew/extend/cachable.rb

```diff
@@ -7,6 +7,11 @@ def cache
     @cache ||= T.let({}, T.nilable(T::Hash[T.untyped, T.untyped]))
   end
 
+  sig { params(hash: T.nilable(T::Hash[T.untyped, T.untyped])).void }
+  def overwrite_cache!(hash)
+    @cache = hash
+  end
+
```

Comment:
Ideally this wouldn't be public otherwise you could do `Formulary.overwrite_cache! {}`

Reply:
Good point!